client-v2/src/lib/dnd/__tests__/dnd.spec.tsx

```diff
@@ -165,16 +165,11 @@ describe('Guration', () => {
     expect(edit).toEqual({
       data: { id: '1' },
       from: { type: 'b', id: '0', index: 0 },
-      to: { id: '2', type: 'a', index: 0 }
+      to: { type: 'a', id: '2',  index: 0 }
     });
 
     runDrag(nodeProps)(dropProps, false);
 
-    expect(edit).toEqual({
-      data: { id: '1' },
-      from: { type: 'b', id: '0', index: 0 },
-      to: { id: '2', type: 'a', index: 1 }
-    });
```

Comment:
Okay. So I'm deleting a test here `allows dropping on to nodes when renderDrop is defined` (2nd assertion) because it's failing. This is why I think it's the right thing to do. 
Please check my reasoning here because I do not fully understand how this test file works. 
The test expected the article to go *below* the story hovered on (index 1) , with our current code, it's going above (index 0).

```
    Expected value to equal:
      {"data": {"id": "1"}, "from": {"id": "0", "index": 0, "type": "b"}, "to": {"id": "2", "index": 1, "type": "a"}}
    Received:
      {"data": {"id": "1"}, "from": {"id": "0", "index": 0, "type": "b"}, "to": {"id": "2", "index": 0, "type": "a"}}
```

Here's why I'm deleting it:
1) the drag and drop behaviour on the UI is working as expected. Whatever this test is failing on, it's not a behaviour that affects the user

2) As I understand @jonathonherbert 's changes to `Level`, [removing the DropIndexOffset method](https://github.com/guardian/facia-tool/pull/804/files#diff-7abf21e9815f013cb90e805bc9fba4e6L172) means that when you hover over a story the dragged article can only ever go on top. Which simplifies both the code and the user interaction. Therefore this test, checking things can go below, is redundant. 

keen to hear any thoughts 
@jonathonherbert @jennygrahamjones

Reply:
@ajwl your explanation is thorough and makes perfect sense. This failing test is a result of a change I made, so I'm sorry you were the person that had to debug it! Removing this test is the correct thing to do. 👍

Reply:
☝️ what @jonathonherbert said!

Reply:
thanks both!